.infrastructure/ingress.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: todoapp-ingress
+  namespace: todoapp
+  annotation:
```

Comment:
There is a typo in the metadata section. The key should be 'annotations' instead of 'annotation'. This is necessary for Kubernetes to recognize multiple annotations correctly.